src/components/NewPageChoice.tsx

```diff
@@ -29,53 +29,63 @@ const NewPageChoice = ({
       onClick();
     }
   }, [disabled, onClick]);
+  const [isFocused, setIsFocused] = useState(false);
```

Comment:
Feels like there should be a way to do it without this.

Reply:
Stateless approach using `_groupFocusWithin` -> [bfc291b](https://github.com/microbit-foundation/ml-trainer/pull/518/commits/bfc291be78c5d7ab96939519fb3973382a310e69)

Reply:
Thanks! Sorry for my totally non-constructive feedback and glad it turned out to be true.